ansible_collections/arista/avd/molecule/eos_validate_state/reports/test_results/dc2-spine2-results.json

```diff
@@ -795,6 +795,14 @@
     ],
     "description": "Verifies the last reload cause of the device."
   },
+  {
```

Comment:
why is this changing?

Reply:
it was optional before (with tags) - it is not anymore. Do you want me to list it as a change?